### ISSUE TITLE
Feature Request: Allow media flyout to be pinned so it's always visible

### ISSUE THREAD

**Issue by bryanniwa** [ModernFlyouts-Community/ModernFlyouts#368]
The media flyout looks great and the controls are very extensive so I would like to be able to pin it so the controls and album info etc. are always available.

I know this replaces the system flyouts so I'm not sure if it is possible to do what I am asking. I would be interested in helping out with this if needed.

**Comment by Tropix126** [comment]
A possible bodge for this would just to set the flyout hide timer to something ridiculously high (can't be too high or else the integer will be converted to scientific notation).

**Comment by Tropix126** [comment]
https://user-images.githubusercontent.com/42101043/104865050-7acc1800-5900-11eb-9e99-74ef2c195a35.mp4

quick proof of concept (999999999ms is 277 hours)

**Comment by Cyberdroid1** [comment]
@bryanniwa Will that work for you? 😄

@ShankarBUS any future plan for this?

**Comment by bryanniwa** [comment]
yeah, seems like a good compromise 😂. i didn't even think of that. The only problem is if you want to dismiss it, you would need to go back and change the time back to default

I know you can have app windows always on top so I wonder how hard it would be to implement this for the flyout

**Comment by bryanniwa** [comment]
ohh, what if a pin button is introduced that sets the timer to something super high and then unpinning puts it back to default?

**Comment by Tropix126** [comment]
My proposal for this is to have two options for users to choose from:
- Autohide (hides flyout after a set interval)
- Toggle Mode (shows flyout indefinetly until media key is pressed agian)

This could just be two radio buttons.
![image](https://user-images.githubusercontent.com/42101043/104872672-1a46d600-5914-11eb-99c4-d17e0b5aeddc.png)

**Comment by ShankarBUS** [comment]
I always knew such situations would occur.

So, I pre-planned things and baked this feature right into the FlyoutWindow code as `IsTimeoutEnabled`. If someone's willing to contribute, the could hook up the property to settings and app data. That's it.

**Comment by bryanniwa** [comment]
oh, I would love to contribute! I'm trying to get a handle of C# a bit and this would be awesome

**Comment by ShankarBUS** [comment]
Awesome!

**Comment by ShankarBUS** [comment]
C# is just Microsoft Java.

You could adapt to it real quick.

XAML may be a bit of hectic.

But you will catch on it sooner or later.

**Comment by bryanniwa** [comment]
ok, thanks sounds good. I have some decent experience with Java. Any recommendations with which workloads to enable in visual studio?

**Comment by ShankarBUS** [comment]
Are you on Discord or Telegram?

I will give you further instructions out of here.

**Comment by bryanniwa** [comment]
yep, I'm on both. Discord is probably better

**Comment by Cyberdroid1** [comment]
> yeah, seems like a good compromise 😂. i didn't even think of that. The only problem is if you want to dismiss it, you would need to go back and change the time back to default

No no, just click the X in right corner of the flyout. Poof!

**Comment by bryanniwa** [comment]
oh true, i didn't see that. I think I'll still work on something here

**Comment by lazurit11** [comment]
I suggest repurpose the current pin button to do that, imo it is useless now (why pin top menu if I chose to hide it?)

**Comment by bryanniwa** [comment]
I like that idea too. I haven't had a chance to work on it yet so if ok, I can do it this way

**Comment by alient12** [comment]
> My proposal for this is to have two options for users to choose from:
> 
> * Autohide (hides flyout after a set interval)
> * Toggle Mode (shows flyout indefinetly until media key is pressed agian)
> 
> This could just be two radio buttons.
> ![image](https://user-images.githubusercontent.com/42101043/104872672-1a46d600-5914-11eb-99c4-d17e0b5aeddc.png)
> 
> This could raise issues with lock keys though. I think if this is added, it should wait for 1.0 or later.

If this option were on the Flyout self as a pin button, It would be great. This option would be annoying for the other Flyouts such as Caps lock Flyout, etc.
